Makefile

```diff
@@ -45,20 +45,36 @@ endif
 
 package: lib-native lib-native-release
 	mkdir -p dist
-	echo "$(GIT_TAG_FULL)" > dist/commit-sha
+	echo "Zipping the binaries ..."
+	echo "$(GIT_TAG_FULL)" > dist/wgpu-native-git-tag
 	for RELEASE in debug release; do \
-		ARCHIVE=$(ARCHIVE_NAME)-$$RELEASE.zip; \
+		ARCHIVEDIR=toarchive; \
+		ARCHIVEFILE=$(ARCHIVE_NAME)-$$RELEASE.zip; \
 		LIBDIR=$(TARGET_DIR)/$$RELEASE; \
-		rm -f dist/$$ARCHIVE; \
+		rm -r -f dist/$$ARCHIVEDIR; \
+		rm -f dist/$$ARCHIVEFILE; \
+		mkdir dist/$$ARCHIVEDIR; \
+		mkdir dist/$$ARCHIVEDIR/include; \
+		mkdir dist/$$ARCHIVEDIR/lib; \
+		cp ./dist/wgpu-native-git-tag         dist/$$ARCHIVEDIR | true; \
+		cp ./ffi/webgpu-headers/webgpu.h      dist/$$ARCHIVEDIR/include | true; \
+		cp ./ffi/wgpu.h                       dist/$$ARCHIVEDIR/include | true; \
+		cp ./$$LIBDIR/libwgpu_native.os       dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/libwgpu_native.dylib    dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/libwgpu_native.a        dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/libwgpu_native.dll.a    dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/wgpu_native.dll         dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/wgpu_native.lib         dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/wgpu_native.dll.lib     dist/$$ARCHIVEDIR/lib | true; \
+		cp ./$$LIBDIR/wgpu_native.pdb         dist/$$ARCHIVEDIR/lib | true; \
```

Comment:
Because many of these files are platform-specific. Can also do some if-triage instead. Yeah, let me change that to triage over different platforms and make it explicit.

Reply:
done